docs/source/examples/distributed-jobs.rst

```diff
@@ -47,3 +48,5 @@ SkyPilot exposes two environment variables to distinguish per-node commands:
   You can retrieve the number of nodes by :code:`echo "$SKY_NODE_IPS" | wc -l`
   and the IP address of the third node by :code:`echo "$SKY_NODE_IPS" | sed -n
   3p`.
+- :code:`SKY_NUM_GPUS_PER_NODE`: number of GPUs reserved on each node (the same
```

Comment:
nit: number of GPUs reserved on each node to execute the task; the same as the count in ``accelerators: <name>:<count>``.



nit: L43 add a period?